source/developer/running-mattermost.rst

```diff
@@ -1,6 +1,11 @@
 Running Mattermost
```

Comment:
This file needs modification. For instance "To check styles of Go and JavaScript" needs to be separate for both repositories.

Reply:
Same question for "Running Mattermost". Is this a document to get a user up-and-running to run the webapp locally with the server? There is a section `To run changes to client-side code:` so I'm just wondering if you have specific thoughts on what's lacking or what's present that should be there?